winlink_monitor.py

```diff
@@ -269,7 +306,7 @@ def send_probe(node):
 
 def poll_for_probe(probe):
     sleep_int = FETCH_RETRY_INTERVAL
-    for i in range(FETCH_RETRIES_COUNT):
+    for _i in range(FETCH_RETRIES_COUNT):
```

Comment:
Where's this from? Is this a python-ism I'm not aware of? 

Googling didn't turn anything up but suggests it may break ipython if the code is loaded there.

Reply:
I was actually in error here for a different reason.  And confused a bit.  _var is conventionally used for variables that should not be access from outside the context.  _ (alone) can be used for a variable that is intentionally going to be ignored (like a function return) and is intended to let lint and other users know that you are aware it returns a value and you are intentionally going to ignore the returned value.